content/tutorials/webrtc/basics/en/index.html

```diff
@@ -197,15 +203,15 @@ <h3 id="toc-constraints">Constraints</h3>
 
 <p>It's also possible to use screen capture as a MediaStream source. This is currently implemented in Chrome using the experimental chromeMediaSource constraint, as in <a href="https://html5-demos.appspot.com/static/getusermedia/screenshare.html" title="Screenshare demo">this demo</a>. This feature is not yet available in Opera. Note that screen capture requires HTTPS.</p>
 
-<p>Chrome apps also now make it possible to share a live 'video' of a single browser tab via the experimental <a href="http://developer.chrome.com/dev/extensions/tabCapture.html" title="chrome.tabCapture API documentation">chrome.tabCapture</a> API. For a screencast, code and more information, see the HTML5 Rocks update: <a href="http://updates.html5rocks.com/2012/12/Screensharing-with-WebRTC" title="HTML5 Rocks update article: Screensharing with WebRTC">Screensharing with WebRTC</a>. This feature is not yet available in Opera.</p>
+<p>Chrome apps also now make it possible to share a live 'video' of a single browser tab via the experimental <a href="http://developer.chrome.com/dev/extensions/tabCapture.html" title="chrome.tabCapture API documentation">chrome.tabCapture</a> API. For screencast, code and more information, see the HTML5 Rocks update: <a href="http://updates.html5rocks.com/2012/12/Screensharing-with-WebRTC" title="HTML5 Rocks update article: Screensharing with WebRTC">Screensharing with WebRTC</a>. This feature is not yet available in Opera.</p>
```

Comment:
I meant that it maybe wasn't worth pointing out that screensharing isn't supported by Opera.

Maybe rephrase: 'It's also possible to use screen capture as a MediaStream source in Chrome using the experimental ...'